cogs/committee_actions_tracking.py

```diff
@@ -585,6 +590,19 @@ async def list_user_actions(
         else:
             action_member = ctx.user
 
+        if committee_role not in ctx.user.roles and action_member != ctx.user:
```

Comment:
```suggestion
        if action_member != ctx.user and committee_role not in ctx.user.roles:
```

Condition check is more legible when flipped.